app/models/searchgov_domain.rb

```diff
@@ -26,7 +27,7 @@ class DomainError < StandardError; end
   attr_readonly :domain
   attr_reader :response
 
-  scope :ok, -> { where(status: GOOD_STATUS) }
+  scope :ok, -> { where(status: INDEXABLE_STATUS) }
```

Comment:
It took me some time to figure this out. I think `ok` and `good` doesn't say much. What do you think about renaming this scope to `indexable` and the next one `no_indexable` instead?